build.sbt

```diff
@@ -2,7 +2,9 @@ name := "scalajs-react-virtuoso"
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
-val reactVirtuoso      = "1.8.6"
+Global / excludeLintKeys += webpackExtraArgs
```

Comment:
I'll pretend I don't see this so I don't have to know what it does. 🙈

Reply:
LOL it just avoids an sbt warning.